packages/fluentui/react/src/themes/teams/components/Table/tableCellStyles.ts

```diff
@@ -25,7 +28,7 @@ export default {
       height: '100%'
     };
   },
-  content: ({ props: { truncateContent } }: ComponentStyleFunctionParam<TableCellProps, TeamsTableVariables>): ICSSInJSStyle => {
+  content: ({ props: { truncateContent } }: ComponentStyleFunctionParam<TableCellStylesProps, TeamsTableVariables>): ICSSInJSStyle => {
```

Comment:
It may be better if you just create a const with the typing and avoid adding typings on each slot, something like:
```
const tableCellStyles: ComponentSlotStylesPrepared<TableCellStyleProps, TeamsTableVariables> = {
  // slots here
}
```